spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -89,7 +89,6 @@
       click_submit_default
 
       expect(current_url).to start_with('http://localhost:7654/auth/result')
-      expect(page.get_rack_session.keys).to include('sp')
```

Comment:
Should we just flip the assertion to expect it's not there? `expect().to_not include('sp')`

Ditto for the one below